app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  # Custom method: Updates the comments counter for a post
+  def update_post_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] [Optional] Same here, It would be better professionally if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) to be specific the `after_save` method to call the `update_post_comments_counter` method every time a new post has been saved to the database, then make the `update_post_comments_counter` method private.